.gitignore

```diff
@@ -19,16 +19,6 @@ out/
 profile.out
 vendor/
 debug.log
-/js/*.db.d
-tool/deployments/*/data
-node_modules
-
-# Go CMD's build output
-go/cmd/berty/berty
-go/cmd/berty-doctor/berty-doctor
-go/cmd/berty-integration/berty-integration
-go/cmd/welcomebot/welcomebot
-go/cmd/rdvp/rdvp
 
 # Various berty outputs
 welcomebot.store
```

Comment:
Can you also remove this? It's about the datastore of the Welcome bot, a cmd on the Berty's repo.

Reply:
Done.